src/product_card-styles/rating-block.css

```diff
@@ -0,0 +1,29 @@
+
+.rating-block {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  font-weight: 400;
+  font-size: 10px;
+  margin-bottom: var(---rate-margin-bottom);
+}
+
+.product-card .rating-block .stars {
+  display: flex;
+}
+
+.product-card .rating-block .stars__star {
+  height: var(--each-star-height);
+  width: var(--each-star-width);
+  background-image: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.product-card .rating-block .stars__star:not(:last-child) {
+  margin-right: var(--each-star-margin-rigth);
+}
+
+.product-card .rating-block .stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("../images/star-active.svg");
+}
```

Comment:
use styles from the "stars" task, for all counts of stars

Reply:
i am using

Reply:
why do i have to add 6 rows of stars if we need only one

Reply:
because we should be able to change the number of stars simply by changing the modifier in the class